packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -50,6 +50,13 @@ class _ClickInfo {
     }
 }
 
+/** @internal */
+class _ClickEvent {
+    public clickInfo: _ClickInfo;
+    public evt: IPointerEvent;
+    public timeoutId: number;
+}

```

Comment:
This can be an interface (will save a bit of code when compiled to UMD). Something like:

```javascript
interface_IClickEvent {
    public clickInfo: _ClickInfo;
    public evt: IPointerEvent;
    public timeoutId: number;
};

//...

const object: _IClickEvent = {
clickInfo
// further elements
}
```

Reply:
Changed to interface